src/lava/lib/optimization/solvers/generic/solver.py

```diff
@@ -19,9 +19,12 @@
 from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
 from lava.proc.dense.models import PyDenseModelFloat
 from lava.proc.dense.process import Dense
-from lava.proc.read_gate.models import ReadGatePyModel
-from lava.proc.read_gate.process import ReadGate
-from lava.lib.optimization.utils.solver_benchmarker import SolverBenchmarker

```

Comment:
SolverBenchmarker is not imported here, but is instantiated in line 103

Reply:
Yes, fixing that, thanks!